org.alloytools.alloy.classic.test/doc/eventadmin.md

```diff
@@ -0,0 +1,205 @@
+
+
+
+interface EventConstants {
```

Comment:
what is this file?  Is this documentation or test?

Reply:
this is gone